recipes/uwebsockets/all/conanfile.py

```diff
@@ -46,6 +43,14 @@ def configure(self):
             raise ConanInvalidConfiguration(
                 "%s requires a compiler that supports at least C++%s" % (self.name, minimal_cpp_standard))
 
+    def requirements(self):
+        self.requires("zlib/1.2.11")
+
+        if self.version == "18.3.0":
+            self.requires("usockets/0.4.0")
+        else:
+            self.requires("usockets/0.7.1")
```

Comment:
```suggestion
        if tools.Version(self.version) >= "19.0.0":
            self.requires("usockets/0.7.1")
        else:
            self.requires("usockets/0.4.0")
```

If for some reason we accept another old version (rare case), when reviewing version bump, we don't see all recipe by default.

Reply:
good point! thanks